docs/getting_started.md

```diff
@@ -42,33 +42,6 @@ Note for macOS users: you may need to [increase the memory available](https://do
 
 ## Initialize the management cluster and install the BringYourOwnHost provider
 
-To initialize Cluster API Provider BringYourOwnHost, `clusterctl` requires the following settings, which should
```

Comment:
Does clusterctl now has BYOH as a default provider?

Reply:
Not as a default, but you don't have to explicitly add it to `clusterctl.yaml`

The changes were made in this PR - https://github.com/kubernetes-sigs/cluster-api/pull/5593 and is part of CAPI `v1.1`